src/app/components/audio-recordings/pages/list/list.component.html

```diff
@@ -0,0 +1,135 @@
+<ng-container *ngIf="rows && !error">
+  <h1>Audio Recordings</h1>
+
+  <p>
+    Displaying <strong>{{ pageSize }}</strong> of
+    <strong>all {{ totalModels }}</strong> audio recordings.
+  </p>
+
+  <ngx-datatable
+    #table
+    bawDatatableDefaults
+    [columnMode]="ColumnMode.force"
+    [columns]="columns"
+    [rows]="rows"
+    [count]="totalModels"
+    [offset]="pageNumber"
+    (page)="setPage($event)"
+    (sort)="onSort($event)"
+  >
+    <ngx-datatable-column name="Recorded">
+      <ng-template let-row="row" let-value="value" ngx-datatable-cell-template>
+        <!-- Use resolver site -->
+        <span
+          *ngIf="site"
+          [ngbTooltip]="site.tzinfoTz ?? 'Unknown timezone'"
+          placement="right"
+        >
+          {{ value }}
+        </span>
```

Comment:
We'll want a time component so we can do this in a consistent way across the site